lighthouse-core/gather/gatherers/accessibility.js

```diff
@@ -50,46 +50,51 @@ function runA11yChecks() {
       'svg-img-alt': {enabled: false},
       'audio-caption': {enabled: false},
     },
-    // @ts-expect-error
-  }).then(axeResults => {
-    // axe just scrolled the page, scroll back to the top of the page so that element positions
-    // are relative to the top of the page
-    document.documentElement.scrollTop = 0;
+  });
+
+  // axe just scrolled the page, scroll back to the top of the page so that element positions
+  // are relative to the top of the page
+  document.documentElement.scrollTop = 0;
+
+  /** @param {import('axe-core/axe').Result} result */
+  const augmentAxeNodes = result => {
+    result.nodes.forEach(node => {
+      // @ts-expect-error - getNodeDetails put into scope via stringification
+      Object.assign(node, getNodeDetails(node.element));
+      // @ts-expect-error - avoid circular JSON concerns
+      node.element = node.any = node.all = node.none = undefined;
+    });
 
+    // Ensure errors can be serialized over the protocol
     // @ts-expect-error
-    const augmentAxeNodes = result => {
+    if (result.error instanceof Error) {
       // @ts-expect-error
-      result.nodes.forEach(node => {
-        // @ts-expect-error - getNodeDetails put into scope via stringification
-        Object.assign(node, getNodeDetails(node.element));
-        // avoid circular JSON concerns
-        node.element = node.any = node.all = node.none = undefined;
-      });
+      result.error = {
+        // @ts-expect-error
+        name: result.error.name,
+        // @ts-expect-error
+        message: result.error.mess,
+        // @ts-expect-error
+        stack: result.error.stack,
+        // @ts-expect-error
+        errorNode: result.error.errorNode,
+      };
+    }
+  };
 
-      // Ensure errors can be serialized over the protocol
-      if (result.error instanceof Error) {
-        result.error = {
-          name: result.error.name,
-          message: result.error.message,
-          stack: result.error.stack,
-          errorNode: result.error.errorNode,
-        };
-      }
-    };
-
-    // Augment the node objects with outerHTML snippet & custom path string
-    axeResults.violations.forEach(augmentAxeNodes);
-    axeResults.incomplete.forEach(augmentAxeNodes);
+  // Augment the node objects with outerHTML snippet & custom path string
+  axeResults.violations.forEach(augmentAxeNodes);
+  axeResults.incomplete.forEach(augmentAxeNodes);
 
-    // We only need violations, and circular references are possible outside of violations
-    axeResults = {
-      violations: axeResults.violations,
-      notApplicable: axeResults.inapplicable,
-      incomplete: axeResults.incomplete,
-      version: axeResults.testEngine.version,
-    };
-    return axeResults;
-  });
+  // We only need violations, and circular references are possible outside of violations
+  return {
+    // @ts-expect-error
```

Comment:
explanations? I'm guessing the previous version just defaulted to `any` or something and got away with it :)